src/UserService.ts

```diff
@@ -91,6 +91,34 @@ class UserService extends Base {
 
     throw new Error(`jwt.preferred_username is ${decodedAccessToken?.preferred_username}`);
   }
+
+  async deleteOne(username: string): Promise<void> {
+    const { apiEndpointAddress, got } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/users/${username}`;
+
+    await throwExpected(
+      () => got.delete(url),
+      {
+        500: error => {
+          if (error?.body.message.includes("Token")) {
```

Comment:
Note: this should never really happen in practise.. but I guess it's okay to handle it here. (It would be programming error of the server for this to happen).

Should we instead return InternalServerException here?

Reply:
Turns out we already have a `TokenNotFoundException` which carries a 500 error code, I'll use it here.